contributing_to_docs/contributing_user_stories.adoc

```diff
@@ -0,0 +1,69 @@
+[[contributing-user-stories]]
+= Contribute user stories to OpenShift documentation
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+:description: Basic information about how to create user stories for OpenShift GitHub repository
+
+toc::[]
+
+== Modularization back story
+OpenShift docs are getting modularized, starting from OpenShift 4.0. Older content will
```

Comment:
"Older content will be rewritten from scratch to conform to our modularization guidelines and new feature content will be created using user stories."

I might say something like "All existing content will be completely replaced with content that is based on user stories and complies with the modularization guidelines. All future content must support a user story."

We're not rewriting *all* the old content, and I don't think it's helpful to continue to think of "feature" doc.